bulletroboy/exoforce.py

```diff
@@ -286,3 +294,122 @@ def get_motors(self):
 
 	def get_tendons(self):
 		return [ muscle.tendon for muscle in self.muscle_units ]
+
+
+class Movements():
```

Comment:
- [ ] create `operator.py`
- [ ] move `Operator` and `Movement` classes to a new file
- [ ] refactor `cage_simulation.py` accordingly

Reply:
I created an operator.py file and passed the operator and movement classes to it. However, when I try running it, it gives me an importerror: cannot import name 'Operator'.

If I change the file_name from operator.py to test.py (or any other name) it works. I searched for possible answers in google that mention circular dependent imports but I checked it and I doubt its the case. Im stuck here. Im considering calling the file: operator_file.py. Would that be okay?

Reply:
no, please dont call it operator_file.py
ask your fellow team members to help you out there

Reply:
Karim and Manuel helped me figure it out. The problem came from the fact that "operator" is an already existing python module. It was fixed by setting the path one folder up and using "from bulletroboy.operator import Operator" :)

Reply:
👍